psi4/src/psi4/libmints/wavefunction.h

```diff
@@ -647,15 +647,30 @@ class PSI_API Wavefunction : public std::enable_shared_from_this<Wavefunction> {
     // Set the external potential
     void set_external_potential(std::shared_ptr<ExternalPotential> external) { external_pot_ = external; }
 
-    /// Get and set variables dictionary
-    double get_variable(const std::string key);
-    void set_variable(const std::string key, double value) { variables_[key] = value; }
-    std::map<std::string, double> variables() { return variables_; }
-
-    /// Get and set arrays dictionary
-    SharedMatrix get_array(const std::string key);
-    void set_array(const std::string key, SharedMatrix value) { arrays_[key] = value; }
-    std::map<std::string, SharedMatrix> arrays() { return arrays_; }
+    /// Get and set variables and arrays dictionaries
+    int has_scalar_variable(const std::string& key);
```

Comment:
I think the intent of the `has_` functions would be  more apparent if they were `bool` instead of `int`